crowdkit/datasets/load_dataset.py

```diff
@@ -11,7 +11,7 @@
 
 
 def load_dataset(dataset: str, data_dir: Optional[str] = None) -> Tuple[pd.DataFrame, pd.Series]:
-    """Downloads a dataset from remote and loads it into Pandas objects.
+    """Downloads a [dataset](https://toloka.ai/datasets) from remote and loads it into Pandas objects.
```

Comment:
This is a very well-placed link as `load_dataset()` downloads data from this place. Please keep it.

Reply:
No, please don't. We're planning to add datasets that are widely used by the community but weren't collected on Toloka. In this case, they won't be listed on https://toloka.ai/datasets and this change won't make sense.

Reply:
Agreed.